migrations/common/input_options.go

```diff
@@ -13,15 +13,5 @@ type InputOptions struct {
 	Tags             map[string]string `toml:"tags,omitempty"`
 
 	// Filter options
-	NamePass       []string            `toml:"namepass,omitempty"`
-	NameDrop       []string            `toml:"namedrop,omitempty"`
-	FieldPassOld   []string            `toml:"pass,omitempty"`
-	FieldPass      []string            `toml:"fieldpass,omitempty"`
-	FieldDropOld   []string            `toml:"drop,omitempty"`
-	FieldDrop      []string            `toml:"fielddrop,omitempty"`
-	TagPassFilters map[string][]string `toml:"tagpass,omitempty"`
-	TagDropFilters map[string][]string `toml:"tagdrop,omitempty"`
-	TagExclude     []string            `toml:"tagexclude,omitempty"`
-	TagInclude     []string            `toml:"taginclude,omitempty"`
-	MetricPass     string              `toml:"metricpass,omitempty"`
+	FilterOptions
 }
```

Comment:
Have you tested this? This does not work when marshalling the data again!

Reply:
The tests seem to succeed? Although it looks like only the one for riemann has an actual test with `namepass` and `fieldpass` in it, and that one also works fine..

Reply:
Would you care to add a testcase to `migrations/inputs_httpjson/testcases`? This migration uses the above structure IIRC.

Reply:
It looks like that is a limitation of the used TOML library (which is kinda outdated as well..)